scripts/config.ts

```diff
@@ -9,8 +9,14 @@ const env = process.env.NODE_ENV || 'development';
 const __DEV__ = env === 'development';
 const __PERF__ = !!process.env.PERF;
 const __PROD__ = env === 'production';
-// This needs a trailing slash or images won't work
-const __BASENAME__ = process.env.DEPLOYBASEPATH ? `/${process.env.DEPLOYBASEPATH}/react-northstar/` : '/';
+let __BASENAME__ = process.env.PR_DEPLOY
+  ? // This needs a trailing slash or images won't work
+    `/pr-deploy-site/${process.env.BUILD_SOURCEBRANCH}/react-northstar/`
+  : '/';
+
+if (process.env.OFFICIALRELEASE) {
```

Comment:
I might not need `process.env.OFFICIALRELEASE` but I need to find out how/where `process.env.PR_DEPLOY` is set

Reply:
using this env variable would still give us an easy way to build docsites for single versions